zebra-utils/Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "zebra-utils"
+version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Utilities for Zebra development, not for library or application users"
 license = "MIT OR Apache-2.0"
-version = "1.0.0-beta.25"
+repository = "https://github.com/ZcashFoundation/zebra"
 edition = "2021"
 
```

Comment:
he Zebra dependencies in this file need versions added.
Do the dev-dependencies need versions as well?

Reply:
Looks like the dev-dependencies do not need versions